lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -41,7 +41,7 @@ function wp_theme_has_theme_json( $clear_cache = false ) {
 	 * Clean theme.json related cached data.
 	 */
 	function wp_theme_clean_theme_json_cached_data() {
-		wp_theme_has_theme_json( true );
+		wp_cache_delete( 'wp_theme_has_theme_json' );
```

Comment:
I figured that using as cache key the method name we have some advantages: 1) searching the codebase also brings this up, which is helpful for discoverability and 2) we won't have conflicts with cache keys by virtue of the PHP namespace.

Reply:
See my above suggestion on using a cache group, I would be in favor of that both for it being a best practice and for eventually having a single group we can cache different block theme aspects in.

Reply:
ok, going to use `theme_json` as group.

As my understanding of groups go, the only value of using relies on potentially using it in `wp_cache_add_non_persistent_groups` or `wp_cache_get_multiple`. Is that correct? You can't reset all keys of a group at once, for example.

Reply:
@oandregal You actually can just since WordPress 6.1, as there is now [`wp_cache_flush_group()`](https://developer.wordpress.org/reference/functions/wp_cache_flush_group/). It requires the cache implementation to adopt it, but the infrastructure is there, so I think that's a good enough reason to go with a group, as it will be helpful in the long run.

Reply:
Brilliant, thanks for sharing.